### ISSUE TITLE
Slices are empty in class SwiftFortuneWheel

### ISSUE THREAD

**Issue by brunodornelesbr** [sh-khashimov/SwiftFortuneWheel#2]
When using SwiftFortuneWheel without storyboard, startAnimating(finishIndex) vanishes with the wheel. I've did a little research and found out that slices are not assigned on to the SwiftFortuneWheel class, only to WheelView. When startAnimating(finishIndex) tries to calculate the sliceDegree, it tries to divide by zero, hence the bug.

I've fixed this in my code by reassigning to SwiftFortuneWheel slices variable 

```
self.fortuneWheel = SwiftFortuneWheel(frame: frame, slices: mappedSlices, configuration: configuration)
self.fortuneWheel.slices = mappedSlices
```

But I think this can be easily be solved by assigning it at the init function of the SwiftFortuneWheel

**Comment by sh-khashimov** [comment]
it was fixed in version 0.6.2. Can you please confirm that?

**Comment by brunodornelesbr** [comment]
I'm on version 0.8.0

**Comment by sh-khashimov** [comment]
Sorry, it was my stupid mistake. Thank you for reporting this.
Now It should be fixed on 0.8.1.

**Comment by brunodornelesbr** [comment]
Thanks for the quick fix
